### DISCUSSION TITLE
How to change the theme of repo card? Please help me:<

### DISCUSSION THREAD

**Discussion by qnhat2004** [anuraghazra/github-readme-stats#3564] [Q&A] [answered]
it likes the title

**Comment by FajarKim** [comment, marked as answer]
@qnhat2004 To add a theme to the repo card, you can use the `theme=THEME_NAME` parameter in the link. You can replace `THEME_NAME` with a list of [available themes](https://github.com/anuraghazra/github-readme-stats/tree/master/themes). For example, I use the `neon` theme:

```markdown
![Repo Card](https://github-readme-stats.vercel.app/api/pin/?username=anuraghazra&repo=github-readme-stats&theme=neon)
```

Result:

![Repo Card](https://github-readme-stats.vercel.app/api/pin/?username=anuraghazra&repo=github-readme-stats&theme=neon)

See full documentation: https://github.com/anuraghazra/github-readme-stats#github-extra-pins

**Reply by qnhat2004** [reply]
Do I need to add a path to the repository? Suppose I have a repository named **Subject_at_University** and I want to change the theme of the repo card to `ambient_gradient`, can you give me an example on how to edit it?

**Reply by FajarKim** [reply]
All you need to do is replace the `username` parameter of `anuraghazra` with your github username and the `repo` parameter of `github-readme-stats` with the name of the repository you want to create the card.

If you want to use your repository `Subject_at_University` with the `ambient_gradient` theme, the link is like this:

```markdown
![Repo Card](https://github-readme-stats.vercel.app/api/pin/?username=qnhat2004&repo=Subject_at_University&theme=ambient_gradient)
```

Result
![Repo Card](https://github-readme-stats.vercel.app/api/pin/?username=qnhat2004&repo=Subject_at_University&theme=ambient_gradient)

**Reply by qnhat2004** [reply]
Should I paste it into the github profile or the repo's readme file? Sorry because i just used it

**Reply by FajarKim** [reply]
It depends on you, whether you want to paste it in your github profile or in the readme repo.

> [!NOTE]
> You can see it directly by pasting the link in your browser. 
> ```markdown 
> https://github-readme-stats.vercel.app/api/pin/?username=qnhat2004&repo=Subject_at_University&theme=ambient_gradient
> ```
> [Click for preview link](https://github-readme-stats.vercel.app/api/pin/?username=qnhat2004&repo=Subject_at_University&theme=ambient_gradient)
